filecoin-proofs/src/bin/paramfetch.rs

```diff
@@ -81,77 +120,162 @@ fn fetch(matches: &ArgMatches) -> Result<()> {
         )));
     }
 
-    let parameter_map = get_parameter_map(&json_path)?;
-    let all_parameter_ids = get_mapped_parameter_ids(&parameter_map)?;
+    let manifest = read_parameter_map_from_disk(&json_path)?;
+    let mut filenames = get_filenames_from_parameter_map(&manifest)?;
+
+    println!("{} files in manifest...", filenames.len());
+    println!();
```

Comment:
can be simplified to `println!("{} files in manifest...\n", filenames.len())`

Reply:
Yeah, this is just cruft code left over from the original implementation.